components/brave_rewards/browser/logging_util.cc

```diff
@@ -0,0 +1,136 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/brave_rewards/browser/logging_util.h"
+
```

Comment:
nit: extra line

Reply:
Following style guide @ https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes, however, I notice our project .h files should be after other libraries .h files where I can fix this as part of https://github.com/brave/brave-browser/issues/9166.

Reply:
I have fixed headers to follow style guide